var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -226,14 +226,12 @@ def cmake_args(self):
 
         return options
 
-    # Delegate in the vtk-m built smoke test
-    def smoke_test(self):
+    def test_smoke_test(self):
+        """Test vtk-m with +examples"""
```

Comment:
Hmm.  I know it's currently clear from the name of the test but I have concerns from a maintainability standpoint.

So I would still prefer the docstring to be a bit more descriptive of what the test is doing, which is build and run the smoke test example.  

What if another test method is later added that also relies on `+examples` but someone forgets to fix the comment here?